argo/common/suite_unit_test.go

```diff
@@ -5,9 +5,10 @@ package common
 import (
 	"encoding/json"
 	"fmt"
+	"testing"
+
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"testing"
 )
 
 func TestCommonUnitSuite(t *testing.T) {
```

Comment:
This test suite doesn't follow the same pattern as all the others, i.e. having just the suite setup in this suite file and then the tests for each component in own unit_test files. So split the existing namespacename tests out into own file and create a run_completion_event_unit_test file

Reply:
done in [6996d38](https://github.com/sky-uk/kfp-operator/pull/352/commits/6996d3803367bab1833668a9763c2db01a8fcc73)